Docker_files/ocsci_container/scripts/edit_yaml.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import yaml
+import sys
+import os
+
+from pathlib import Path
+
+cli_args = sys.argv[1:]
+
+
+def send_cmd(cmd=None):
+    return subprocess.check_output(cmd, shell=True, universal_newlines=True)
+
+
+def yaml_to_dict(path=None):
```

Comment:
the `path` parameter seems to be mandatory, so I would remove the default value `None` here.

Reply:
done